.github/workflows/docker.yml

```diff
@@ -2,7 +2,7 @@ name: Docker image
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, task/springboot-module ]
```

Comment:
Was adding `task/sprintboot-module` only to test this PR? Or is this will be used after the branch is merged?

Reply:
Oops, it was to test this PR. Removed in https://github.com/MobilityData/gtfs-validator/pull/962/commits/36fe8afabf1bd2d2b023c8df077e010b7fe3edf0.